dronekit_sitl/__init__.py

```diff
@@ -103,7 +117,14 @@ def download(system, version, target, verbose=False):
         if not os.path.isdir(sitl_target):
             os.makedirs(sitl_target)
 
-        testfile = urllib.URLopener()
+        if six.PY3:
```

Comment:
I don't have experience working with `six` but from the docs seems like using `six.moves.urllib` would be a better approach here, what do you think?